src/main/java/uk/gov/digital/ho/hocs/aws/LocalStackConfiguration.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.digital.ho.hocs.aws;
+
+import com.amazonaws.ClientConfiguration;
+import com.amazonaws.Protocol;
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.AmazonSQSClientBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+@Configuration
+@Profile({"local"})
+public class LocalStackConfiguration {
+
+    public static final String REGION = "eu-west-2";
```

Comment:
You hard code this here, but we do have it in the application properties?

Reply:
done